_events/2025/2025-10-community-call.md

```diff
@@ -0,0 +1,22 @@
+---
+title: October 2025 Community Call
+expires: 2025-10-10
+event_date: "October 10, 2025"
+layout: event
+duration: 60
+repeated: false
+category: community-call
+time:
+    - - start: 2025-10-10T19:00:00Z
+        end: 2025-10-10T20:00:00Z
+---
+
+On October 10, it’s time for our monthly community call! This one falls right after USRSE’25, so many of us will be just returning from an exciting week of travel, talks, and connections. Instead of a formal program, we’re keeping things light — come join us for a relaxed virtual hangout with fellow RSEs, swap conference highlights, and simply unwind together. No agenda, just open conversation and community connection! And if you weren’t able to attend the conference, this is a great chance to hear impressions and stories directly from your colleagues. 
+
+The typical structure of a community call starts with an overview of recent US-RSE updates, followed by short presentations. After the presentations we will break out into smaller groups in which we discuss the call topic. Everyone is welcome to join the discussion or just hang back and listen in. If you prefer just to listen in, that is perfectly fine, but we encourage you to share your opinions and thoughts as you feel comfortable. If you want to switch to another group, this is also fine. Just go back to the main room and let the call organizers know.
```

Comment:
```suggestion
```

@jdamerow and @roelofsaj - Should we remove this section as it's going to be different this time?